src/app/story/storyrow/storyrow.component.html

```diff
@@ -1,6 +1,7 @@
 <div class="node-uid-column story-row-column">
   <div class="node-display-name">{{ node.display_name | nodeDisplayName | truncate: 30 }}</div>
   <div class="node-timestamp">{{ node.timeline_timestamp }}</div>
+  <div class="node-timestamp" *ngIf="!node.timeline_timestamp">No timestamp information available</div>
```

Comment:
Won't two divs be present in this case? You should use `*ngIf` on the other div as well, but use an "else" for this particular one.

Reply:
It doesn't show up as null so you don't see anything, but I agree it'd be nicer code-wise